src/compositions/useLink.ts

```diff
@@ -30,6 +32,10 @@ export type UseLink = {
    * True if route matches current URL. Route is the same as what's currently stored at `router.route`.
    */
   isExactMatch: Ref<boolean>,
+  /**
+  * Returns true if host does not match host stored on router instance
+  */
+  isExternal: Ref<boolean>,
```

Comment:
This is unrelated to this PR but I moved this here to useLink from router link. Keeping the link logic in the composition.